index.php

```diff
@@ -90,6 +101,30 @@
 
 
     ?>
+
+    <?php
+
+    $validDelete = true;
+    if (isset($_POST['id'])) {
+        $inputtedId = $_POST['id'];
+        if (strlen($inputtedId) == 0) {
+            echo "no ID inputted";
+            $validDelete = false;
+        } elseif (strlen($inputtedId) > 4) {
+            echo "ID is too long";
+            $validDelete = false;
+        }
+    }
+
+    var_dump($_POST);
+    if ($validDelete == true) {
+        $removeMonitor = new MonitorModels($db);
+        $removeMonitor->removeMonitor($inputtedId);
+    } else {
+        echo "Not all info correct to delete";
```

Comment:
I'm hapy with everything, apart from how the error messages are working at the moment. The messages are good, but you see the adding error message even when youre not trying to add. Before merging this I think it would be worth getting that sorted